### ISSUE TITLE
Prod MBlast Error

### ISSUE THREAD

**Issue by Foxcapades** [VEuPathDB/service-multi-blast#86]
```
javax.ws.rs.InternalServerErrorException: HTTP 500 Internal Server Error
        at org.veupathdb.service.multiblast.service.http.Util.wrapException(Util.java:16) ~[service.jar:1.0.0]
        at org.veupathdb.service.multiblast.service.http.JobService.getJob(JobService.java:114) ~[service.jar:1.0.0]
        at org.veupathdb.service.multiblast.controller.JobController.getJob(JobController.java:87) ~[service.jar:1.0.0]
        at jdk.internal.reflect.GeneratedMethodAccessor25.invoke(Unknown Source) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:564) ~[?:?]
        at org.glassfish.jersey.server.model.internal.ResourceMethodInvocationHandlerFactory.lambda$static$0(ResourceMethodInvocationHandlerFactory.java:52) ~[service.jar:1.0.0]
        at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher$1.run(AbstractJavaResourceMethodDispatcher.java:124) ~[service.jar:1.0.0]
        at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.invoke(AbstractJavaResourceMethodDispatcher.java:167) ~[service.jar:1.0.0]
        at org.glassfish.jersey.server.model.internal.JavaResourceMethodDispatcherProvider$ResponseOutInvoker.doDispatch(JavaResourceMethodDispatcherProvider.java:176) ~[service.jar:1.0.0]
        at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.dispatch(AbstractJavaResourceMethodDispatcher.java:79) ~[service.jar:1.0.0]
        at org.glassfish.jersey.server.model.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:475) ~[service.jar:1.0.0]
        at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:397) ~[service.jar:1.0.0]
        at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:81) ~[service.jar:1.0.0]
        at org.glassfish.jersey.server.ServerRuntime$1.run(ServerRuntime.java:255) [service.jar:1.0.0]
        at org.glassfish.jersey.internal.Errors$1.call(Errors.java:248) [service.jar:1.0.0]
        at org.glassfish.jersey.internal.Errors$1.call(Errors.java:244) [service.jar:1.0.0]
        at org.glassfish.jersey.internal.Errors.process(Errors.java:292) [service.jar:1.0.0]
        at org.glassfish.jersey.internal.Errors.process(Errors.java:274) [service.jar:1.0.0]
        at org.glassfish.jersey.internal.Errors.process(Errors.java:244) [service.jar:1.0.0]
        at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:265) [service.jar:1.0.0]
        at org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:234) [service.jar:1.0.0]
        at org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:680) [service.jar:1.0.0]
        at org.glassfish.jersey.grizzly2.httpserver.GrizzlyHttpContainer.service(GrizzlyHttpContainer.java:356) [service.jar:1.0.0]
        at org.glassfish.grizzly.http.server.HttpHandler$1.run(HttpHandler.java:200) [service.jar:1.0.0]
        at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.doWork(AbstractThreadPool.java:569) [service.jar:1.0.0]
        at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.run(AbstractThreadPool.java:549) [service.jar:1.0.0]
        at java.lang.Thread.run(Thread.java:832) [?:?]
Caused by: java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (USERLOGINS5.MB_UQ_USER_JOB) violated

        at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:509) ~[service.jar:1.0.0]
        at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:461) ~[service.jar:1.0.0]
        at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1104) ~[service.jar:1.0.0]
        at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:553) ~[service.jar:1.0.0]
        at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:269) ~[service.jar:1.0.0]
        at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:655) ~[service.jar:1.0.0]
        at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:270) ~[service.jar:1.0.0]
        at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:91) ~[service.jar:1.0.0]
        at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:970) ~[service.jar:1.0.0]
        at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1205) ~[service.jar:1.0.0]
        at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3666) ~[service.jar:1.0.0]
        at oracle.jdbc.driver.T4CPreparedStatement.executeInternal(T4CPreparedStatement.java:1426) ~[service.jar:1.0.0]
        at oracle.jdbc.driver.OraclePreparedStatement.execute(OraclePreparedStatement.java:3778) ~[service.jar:1.0.0]
        at oracle.jdbc.driver.OraclePreparedStatementWrapper.execute(OraclePreparedStatementWrapper.java:1081) ~[service.jar:1.0.0]
        at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94) ~[service.jar:1.0.0]
        at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94) ~[service.jar:1.0.0]
        at io.vulpine.lib.query.util.query.PreparedWriteQuery.executeStatement(PreparedWriteQuery.java:31) ~[service.jar:1.0.0]
        at io.vulpine.lib.query.util.query.PreparedWriteQuery.executeStatement(PreparedWriteQuery.java:11) ~[service.jar:1.0.0]
        at io.vulpine.lib.query.util.query.QueryImpl.execute(QueryImpl.java:68) ~[service.jar:1.0.0]
        at mb.lib.db.insert.InsertUserQuery.run(InsertUserQuery.java:22) ~[service.jar:1.0.0]
        at mb.lib.db.JobDBManager.linkUserToJob(JobDBManager.java:141) ~[service.jar:1.0.0]
        at org.veupathdb.service.multiblast.service.http.JobService.getJob(JobService.java:75) ~[service.jar:1.0.0]
        ... 26 more
```

**Comment by aurreco-uga** [comment]
in 2022 we had this happening 2 to 8 times a month

**Comment by aurreco-uga** [comment]
this seems addressed in #147 and advise to "ignore this situation and not throw the 500.. since it is harmless..."
